buffalo/algo/als.py

```diff
@@ -98,7 +98,10 @@ def _get_topk_recommendation(self, rows, topk, pool=None):
         return zip(rows, topks)
 
     def _get_most_similar_item(self, col, topk, pool):
-        return super()._get_most_similar_item(col, topk, self.Q, self.opt._nrz_Q, pool)
```

Comment:
I think it is better to use super().method() instead of self.method() for many reasons, check [this](https://stackoverflow.com/questions/50658942/whats-the-difference-between-super-method-and-self-method) out.